js/components/items-container.tsx

```diff
@@ -365,18 +364,8 @@ class ItemsContainer extends React.Component<IComponentProps, IItemsContainerSta
                 objectData
             ) {
                 const page = (state.page - 1) || 0;
-                const paginationSize = parseInt(model.attributes.paginationSize, 10);
 
-                limit = parseInt(
-                    manywho.settings.flow(
-                        `paging.${model.componentType.toLowerCase()}`, this.props.flowKey,
-                    ) || 10,
-                    10,
-                );
-
-                if (!isNaN(paginationSize)) {
-                    limit = paginationSize;
-                }
+                limit = this.getPageSize(model, this.props.flowKey);
```

Comment:
Could limit just be a constant and assigned immediately where it's currently defined?

Reply:
It would seem so. Fixed.